sphinx/source/docs/user_guide.rst

```diff
@@ -3,12 +3,12 @@
 User Guide
 ==========
 
-This user guide is intended to guide you through many common tasks that
+This user guide is intended to walk you through many common tasks that
 you might want to accomplish using Bokeh. The guide is arranged by
 topic:
 
 :ref:`userguide_quickstart`
-    Quickstart guide to Bokeh
+    Read the Quickstart guide to Bokeh to get up and running fast.
```

Comment:
In this list, all the other links/link descriptions (aside from the other one I've fixed) begin with a verb and have that construction. I've changed this to match.